core/qt-ble.cpp

```diff
@@ -57,17 +57,36 @@ void BLEObject::serviceStateChanged(QLowEnergyService::ServiceState s)
 
 void BLEObject::characteristcStateChanged(const QLowEnergyCharacteristic &c, const QByteArray &value)
 {
-	Q_UNUSED(c)
+	if (device_is_hw(device)) {
+		if (c.uuid() == hwAllCharacteristics[HW_OSTC_BLE_DATA_TX]) {
+			receivedPackets.append(value);
+		} else {
+			qDebug() << "ignore packet from" << c.uuid() << value.toHex();
+		}
+	} else {
+	    receivedPackets.append(value);
```

Comment:
What to you exactly mean here? curly braches around one statement? tabs vs spaces?